src/Entity/FenceReviewCase.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\FenceReviewCaseRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass=FenceReviewCaseRepository::class)
+ */
+class FenceReviewCase extends CaseEntity
+{
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $complainant;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $complainantAddress;
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $complainantZip;
+
+    /**
+     * @ORM\Column(type="string", length=255, options={"default": "submitted"})
+     */
+    private $caseState = 'submitted';
+
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $complainantCPR;
```

Comment:
There is a lot of properties that is duplicated on all case entity classes. Should they be moved to the CaseEntity class?

Reply:
Moved shared properties to CaseEntity class